### PR TITLE
Document config-formatter in tools README.md

### PR DESCRIPTION
**Pull request by MakisH**
## Main changes of this PR

Adds `config-formatter` to the `README.md` of `tools/formatting`.

## Author's checklist

* [ ] I used the [`pre-commit` hook](https://precice.org/dev-docs-dev-tooling.html#setting-up-pre-commit) to prevent dirty commits and used `pre-commit run --all` to format old commits.
* I added a changelog file with `make changelog` if there are user-observable changes since the last release.
* I added a test to cover the proposed changes in our test suite. -> N/A
* I sticked to C++17 features. -> N/A
* I sticked to CMake version 3.16.3. -> N/A
* [ ] I squashed / am about to squash all commits that should be seen as one.

## Reviewers' checklist

* @fsimonis is there any reason this was not documented here? I see that we have it in https://precice.org/community-contribute-to-precice.html#optional-help-us-with-some-checks

### CODE REVIEW

File: tools/formatting/README.md [hunk 1]
```diff
@@ -12,6 +12,10 @@ This bash-script checks the format of every c(pp) & h(pp) file in the current an
 It returns 0 if everything is formatted correctly.
 Otherwise, it displays the list of files that do not match the format and returns 1.
 
+## cofig-formatter
```
Comment (uekerman) [inline tools/formatting/README.md:15]:
```suggestion
## config-formatter
```

**Comment by fsimonis** [comment]
Ah I didn't remove the script after moving it to the pre-commit repo.

**Comment by MakisH** [comment]
> Ah I didn't remove the script after moving it to the pre-commit repo.

What does this mean then? Merge or close?

**Comment by uekerman** [comment]
and adjust the documentation on the website

**Pull request closed without merging**